README.md

```diff
@@ -22,11 +22,12 @@ df = DataFrame(species=["E. coli", "B. fragilis", "L. casei"],
                sample2=[0.3, 0.7, 0.0],
                sample3=[0.0, 0.2, 0.8])
 
-abund = AbundanceTable(df)
+abund = abundancetable(df)
 
 @show abund
-@show abund.samples
-@show abund.features
+full(abund)
```

Comment:
The `show` function for `ComMatrix` is different, and optimised to showing larger matrices. `full` returns a dense copy of the abundance table (which is sparse in `ComMatrix`), giving the same functionality as `show` on the AbundanceTable type.